api/spec/src/billing/profile.tsp

```diff
@@ -78,25 +78,55 @@ interface Profiles {
 enum ProfileOrderBy {
   createdAt: "createdAt",
   updatedAt: "updatedAt",
+  default: "default",
+  name: "name",
 }
 
 /**
  * Profile represents a billing profile
  */
 @friendlyName("BillingProfile")
 model Profile {
-  ...global.Resource;
+  ...OmitProperties<global.Resource, "updatedAt">;
+
+  /**
+   * When the resource was last updated.
+   *
+   * For updates this field must be set to the last update time to detect conflicts.
+   */
+  @summary("Last update time of the resource")
+  @visibility("read", "query", "update")
+  updatedAt: DateTime;
 
   @summary("The name and contact information for the supplier this billing profile represents")
   supplier: Invoices.Party;
 
   @summary("The billing workflow settings for this profile")
   workflow: Workflow;
 
+  @summary("The applications used by this billing profile")
+  @visibility("read", "query")
+  apps: ProfileApps;
+
   @summary("Is this the default profile?")
   default: boolean;
 }
 
+/**
+ * ProfileApps represents the applications used by a billing profile
+ */
+@friendlyName("BillingProfileApps")
```

Comment:
Yes, but based on notifications and other parts of the code we are better off adding friendlyName.

For example in this case the auto-generated name would be OpenMeterBilling_ProfileApps, but we don't need the openmeter prefix.

Or in case of the AppType that would become OpenMeterApp_AppType. If we drop the OpenMeter form the namespace we should be able to get aways with some renames and not having friendly name IMHO, but for the time being I would prefer to keep the friendlyNames. cc @tothandras this is something to think about.